module/VuFind/src/VuFind/Search/UrlQueryHelper.php

```diff
@@ -309,6 +309,21 @@ public function replaceTerm($from, $to, $normalizer = null)
         return new static($this->urlParams, $query, $this->config);
     }
 
+    /**
+     * Format a set of facet parameters into a filter string.
+     *
+     * @param string $field    Facet field
+     * @param string $value    Facet value
+     * @param string $operator Facet type to add (AND, OR, NOT)
+     *
+     * @return string
+     */
+    public function formatFilterString(string $field, string $value, string $operator = 'AND'): string
+    {
+        $prefix = ($operator == 'NOT') ? '-' : ($operator == 'OR' ? '~' : '');
```

Comment:
Thanks, @rominail, I like your suggestion! You can use the "Add a suggestion" button in the toolbar of the comment editor to suggest fixes like this directly, like:

```suggestion
        $prefix = match ($operator) {
            'NOT' => '-',
            'OR'  => '~',
            default => '',
        };
```

...this allows the suggestion to be directly committed from within the PR! I'm going to do that now. :-)